lib/jellyfish_web/socket.ex

```diff
@@ -19,33 +19,41 @@ defmodule JellyfishWeb.Socket do
     with {:ok, peer_id} <- Map.fetch(state.params, "peer_id"),
          {:ok, room_id} <- Map.fetch(state.params, "room_id"),
          {:ok, room_pid} <- RoomService.find_room(room_id),
-         :ok <- Logger.metadata(room_id: room_id),
          {:ok, :disconnected} <- Room.get_peer_connection_status(room_pid, peer_id) do
       state =
         state
+        |> Map.put(:room_id, room_id)
         |> Map.put(:peer_id, peer_id)
         |> Map.put(:room_pid, room_pid)
 
-      Logger.info("WebSocket connection from peer #{inspect(peer_id)} accepted")
+      Logger.info("""
+        WebSocket connection from peer #{inspect(peer_id)} accepted, \
+        room #{inspect(room_id)}
+      """)
```

Comment:
I am not sure whether these spaces are visible in logs but in general we write (at least in the engine):

```suggestion      
      Logger.info("""
      WebSocket connection from peer #{inspect(peer_id)} accepted, \
      room #{inspect(room_id)}
      """)
```